fire/SFMainMod.F90

```diff
@@ -94,13 +97,15 @@ subroutine fire_model( currentSite, bc_in)
     if( hlm_use_spitfire == itrue )then
        call fire_danger_index(currentSite, bc_in)
        call wind_effect(currentSite, bc_in) 
-       call charecteristics_of_fuel(currentSite)
-       call rate_of_spread(currentSite)
+       call characteristics_of_fuel(currentSite, MEF, fuel_moisture)
+       call rate_of_spread(currentSite, MEF, fuel_moisture)
        call ground_fuel_consumption(currentSite)
        call fire_intensity(currentSite)
        call area_burnt(currentSite)
        call crown_scorching(currentSite)
        call crown_damage(currentSite)
+       call rate_of_spread(currentSite, MEF, fuel_moisture)
+       ! TODO Recalculate area_burnt, FI for altered Scorch Height (SH)
```

Comment:
Here in subr. `fire_model` I have changed two things:
1) Pass _MEF_ and _fuel_moisture_ **out of** subr. `characteristics_of_fuel` and **into** subr. `rate_of_spread`
2) Call `rate_of_spread` a second time, once we're done with subr. `crown_damage`